### PR TITLE
BigQuery sql syntax update

### PR DESCRIPTION
**Pull request by rathorevaibhav**
## Summary

Syntax update for BigQuery for UNNEST method

## Test Plan

- [x] Tested locally
- [ ] Tested on staging
- [ ] Tested on production

### CODE REVIEW

File: schema/Session.js [hunk 1]
```diff
@@ -139,9 +139,8 @@ cube(`GroupedSessionRetention`, {
           SELECT
             session_id,
             plio_id,
-            user_id,
-            UNNEST(string_to_array(retention, ','))::int AS retention_array
-          FROM ${GroupedSession.sql()} AS session
+            user_id
+          FROM ${GroupedSession.sql()} AS session, UNNEST(SPLIT(session.retention, ',')) AS retention_array
```
Comment (dalmia) [inline schema/Session.js]: if we are making this change - then, we cannot say that we support RDS too - it has to be BQ only, right?

Comment (rathorevaibhav) [reply schema/Session.js]: yeah right. The query is still failing, I'm checking again.

File: schema/Session.js [hunk 2]
```diff
@@ -80,7 +80,7 @@ cube(`GroupedSession`, {
         session.retention,
         ROW_NUMBER() OVER(PARTITION BY session.user_id, session.plio_id
                           ORDER BY session.watch_time DESC) AS rank
-        FROM "public"."session"
+        FROM public.session
```
Comment (dalmia) [inline schema/Session.js]: this is not a change that made it compatible right? If not, let's revert for maintaining consistency or change others too?

Comment (rathorevaibhav) [reply schema/Session.js]: Oh this was failing in BigQuery. It was giving an error for unexpected double quotes. So I had to remove them.

**Pull request merged**